solr/core/src/java/org/apache/solr/handler/admin/CollectionsHandler.java

```diff
@@ -419,8 +418,46 @@ private static void addStatusToResponse(NamedList<Object> results, RequestStatus
      */
```

Comment:
Looking at the above comment, I'm not sure this is accurate anymore. not seeing the "template collection" feature, and I think the "server list" bit has been obsoleted by the replica placement policy props also this does go to the overseer queue subsequently...